docs/UserGuide.md

```diff
@@ -130,25 +130,36 @@ Edits an existing order in the order list.
 Format: `edito INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/ITEM_NAME] [q/QUANTITY]`
```

Comment:
You can consider doing replacing `[i/ITEM_NAME] [q/QUANTITY]` with `[i/ITEM_NAME q/QUANTITY]` to show that they should be entered together, but are still optional

Reply:
good idea! will implement this change :D